languages/translations/ro.json

```diff
@@ -92,37 +92,37 @@
         },
         "continue": {
             "default": {
-                "default": "Virează {modifier}",
+                "default": "Virați {modifier}",
                 "name": "Virați {modifier} pe {way_name}",
                 "destination": "Virați {modifier} spre {destination}",
                 "exit": "Virați {modifier} pe {way_name}"
             },
             "straight": {
                 "default": "Mergeți înainte",
-                "name": "Continuați înainte pe {way_name}",
+                "name": "Mergeți înainte pe {way_name}",
                 "destination": "Continuați spre {destination}",
-                "distance": "Continuați înainte {distance}",
-                "namedistance": "Continuați pe {way_name} {distance}"
+                "distance": "Mergeți înainte pentru {distance}",
+                "namedistance": "Continuați pe {way_name} pentru {distance}"
             },
             "sharp left": {
-                "default": "Virați brusc stânga",
-                "name": "Virați brusc stânga pe {way_name}",
-                "destination": "Virați brusc stânga spre {destination}"
+                "default": "Virați puternic la stânga",
+                "name": "Virați puternic la stânga pe {way_name}",
+                "destination": "Virați puternic la stânga spre {destination}"
             },
             "sharp right": {
-                "default": "Virați brusc dreapta",
-                "name": "Virați brusc stânga pe {way_name}",
-                "destination": "Virați brusc dreapta spre {destination}"
+                "default": "Virați puternic la dreapta",
+                "name": "Virați puternic la dreapta pe {way_name}",
+                "destination": "Virați puternic la dreapta spre {destination}"
```

Comment:
These translations need to be changed [at Transifex](https://www.transifex.com/project-osrm/osrm-text-instructions/) before they can be changed here. Otherwise, the changes would be overwritten the next time we update this project.

Reply:
Is there any way to import the json into Transifex?

Reply:
There is, but it looks like @mdomnita modified some strings in the meantime, so it might be more prudent to merge the values by hand rather than overwriting what’s there. I’ve added you to the Romanian translation team so you can see what options are available. If you’re comfortable using a merge or diff tool on your computer, then you could export the strings, merge the changes, and reimport. But I’ll leave it up to you and @mdomnita as to the specific translations to use.